### ISSUE TITLE
Missing documentation for long running scripts

### ISSUE THREAD

**Issue by phm46** [logtail/monolog-logtail#21]
Hi Better Stack team,
I had problems with logtail handler, as I wasn't able to make it work in my microservices. However, after a while, when I was looking in code, I've noticed, that it uses buffer handler. What I've discovered is, that buffer handler sends logs by default ONLY when `$logger` is closed using `close()` function. This may work in traditional php scripts which are closed at the end of request processing, but for long running php processes it's not usable, because logger is opened for a long time.

This behaviour is not documented anywhere (or at least I wasn't able to find it). Correct way to use logtail handler in long running php scripts is, to adjust parameters of handler. In this case to set `bufferLimit` and `flushOnOverflow`. This will cause to send every X messages based on `bufferLimit`. 

Example of correct way to use logtail handler in long running scripts:
```php
<?php
require "../vendor/autoload.php";

use Monolog\Logger;
use Logtail\Monolog\LogtailHandler;

$logger = new Logger("example-app");
$logger->pushHandler(
      new LogtailHandler(
          "$SOURCE_TOKEN",
          $this->log_level,
          true,
          \Logtail\Monolog\LogtailClient::URL,
          5, //will send logs to Better Stack every fifth log record. You can adjust this to minimise traffic if your script produces lot of messages.
          true //send messages instead of removing old messages
      ));
```

It will be good to update documentation for long running scripts.
Thanks

**Comment by curusarn** [comment]
Hi @phm46!

Thank you for raising this! 
I'll make sure we update our docs and instructions soon.
Thank you for sharing the example configuration!

**Comment by PetrHeinz** [comment]
Hello @phm46 and thanks for raising this issue 🙌 

In the newly released `v3.2.0` (and `v2.2.0`) we've tweaked the default values so this should be an issue anymore. Logs are automatically flushed after 1000 records or 5s (to allow for resolving a regular request before sending any logs over). You can of course customize this behavior to suit your specific needs.

The customization should now be easier with the help of `LogtailHandlerBuilder` with a nice auto-completion support for any modern IDE.

We've also documented a tip for ensuring all logs are sent when a script execution is terminated by CTRL+C, as the doesn't trigger regular shutdown handlers: https://betterstack.com/docs/logs/php/#logging-from-console-scripts


Hope this helps to address logging in long-running scripts 🚀

Please let me know if you have any questions or feedback around these changes, and thanks again for reporting it!
